cppcache/include/geode/TypeRegistry.hpp

```diff
@@ -44,7 +44,7 @@ class APACHE_GEODE_EXPORT TypeRegistry {
    * registered, or there is an error in registering the type; check errno
    * for more information in the latter case.
    */
-  void registerType(TypeFactoryMethod creationFunction);
+  void registerType(TypeFactoryMethod creationFunction, uint32_t id);
```

Comment:
Why an `uint32_t`? Isn't the Java API using a signed int?

Reply:
The lower level type (SerializationRegistry.addType) is indeed an int32_t. Not sure why we had a uint here. Good catch.

Reply:
that's a mulligan